pkg/app/piped/cloudprovider/kubernetes/diff_test.go

```diff
@@ -184,3 +186,43 @@ func TestDiffByCommand(t *testing.T) {
 		})
 	}
 }
+
+func TestDiff(t *testing.T) {
+	testcases := []struct {
+		name      string
+		manifests string
+		expected  string
+		diffNum   int
+	}{
+		{
+			name:      "secret no diff",
+			manifests: "testdata/diff_secret.yaml",
```

Comment:
I recommend not to use `testdata/` file because it seems to have some disadvantages such as readability.
Hence, we can write manifests into table cases directly.
But it've already used in the another test and it's up to you.

Reply:
I'll change to write manifest directly, thank you!